.env.dist

```diff
@@ -31,14 +32,21 @@ WORKFLOW_ID=1
 # Number Insight
 INSIGHT_NUMBER='NUMBER_FOR_INSIGHT'
 
+# Meetings
+ROOM_DISPLAY_NAME=
+ROOM_ID='THE_ROOM_UUID'
+EXPIRATION_DATE='DATE IN ISO 8601 FORMAT'
+THEME_ID=
+SESSION_ID=
```

Comment:
is it worth adding a prefix to these variables to prevent future conflicts? eg. Video might have a session_id.

Reply:
I thought this too, I was using the names defined on page and in keeping with the other language implementations, e.g. on this page https://developer.vonage.com/en/meetings/code-snippets/recording-management/list-recordings?source=meetings. If you think it's worth doing, I can add this anyway.

Reply:
Ah, I see. Makes sense.